src/app/payment/paymentMethod/DigitalRiverPaymentMethod.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useCallback, useContext, FunctionComponent } from 'react';
+import { Omit } from 'utility-types';
+
+import { connectFormik, ConnectFormikProps } from '../../common/form';
+import { FormContext } from '../../ui/form';
+import PaymentContext from '../PaymentContext';
+import { PaymentFormValues } from '../PaymentForm';
+
+import HostedWidgetPaymentMethod, { HostedWidgetPaymentMethodProps } from './HostedWidgetPaymentMethod';
+
+export type DigitalRiverPaymentMethodProps = Omit<HostedWidgetPaymentMethodProps, 'containerId'> & ConnectFormikProps<PaymentFormValues>;
+
+export enum DigitalRiverClasses {
+    base =  'DRElement',
```

Comment:
This is the right class?? I think you need to get the classes from CC common fields, and the same applies for the others enum (complete, empty, focus, invalid, webkitAutofill)